spec/migrations/20180719162710_add_owner_and_group_to_auth_spec.rb

```diff
@@ -0,0 +1,63 @@
+require_migration
+
+describe AddOwnerAndGroupToAuth do
+  class User < ActiveRecord::Base; end
```

Comment:
We don't usually define AR::Base classes in the specs.  Does this wind up nested under some kind of dynamically created RSpec ExampleGroup class?  If not, these should probably be moved into the migration and use a `migration_stub` here to avoid namespace collision.